### PR TITLE
[V0.2]Update user guide

### PR DESCRIPTION
**Pull request by SukiTsang**

### CODE REVIEW

**Review by Skaty — APPROVED** [review]
:+1: @advaypal, please check!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
    * **`add`**`meeting`: adds a task named CS2103 Tutorial
    * **`find`**`meeting `: searches the task named tutorial   
    * **`delete`**`presentation`: delete a task named Quiz
-   * **`update`**`presentation ddate/10/10/2016:1200` : updates presentation to have a deadline on 10/10/2016 on 12:00
+   * **`update`**`1 presentation c/10/10/2016:1200` : updates first task on the list to presentation having a deadline on 10/10/2016 on 12:00
```
Comment (advaypal) [inline docs/UserGuide.md]: What's the 1 for?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -39,29 +39,31 @@
...
-* `add meeting sdate/01-10-2016:1300 ddate/01-10-2016:2200`<br> Adds a meeting event which start on 1-10-2016 at 1 p.m. and ends at 10 p.m.
+* `add proposal c/10-10-2016:2100` <br> Adds a proposal task with a deadline on 10-10-2016 at 21:00
+* `add meeting s/01-10-2016:1300 c/01-10-2016:2200`<br> Adds a meeting event which start on 1-10-2016 at 1 p.m. and ends at 10 p.m.
 * `add shopping` <br> Adds a floating task named revision test which has not specify the start and end date
+* `add tutorial t/cs2103` <br> Adds a flaoting task named tutorial with a tag CS2013
+* `add quiz t/cs2102 t/easy` <br> Adds a flaoting task named tutorial with a tag CS2012 and easy
 
 #### Deleting a task : `delete`
-Deletes a specific task with the same input task name or index from the to-do list.<br>
+Deletes a specific task by task name or index from the to-do list.<br>
```
Comment (advaypal) [inline docs/UserGuide.md:65]: Guys, what if the task name is a number?

File: docs/UserGuide.md (hunk 1 above)
Comment (advaypal) [inline docs/UserGuide.md]: should specify that the number is the index number

Comment (SukiTsang) [reply docs/UserGuide.md]: ok

File: docs/UserGuide.md (hunk 2 above)
Comment (Skaty) [reply docs/UserGuide.md:65]: That's a problem. While we can restrict it such that a number (or a string of it) cannot be a task, we have to also take into account how restricted we want the task name to be. :)

Comment (Skaty) [reply docs/UserGuide.md:65]: @advaypal: My recommendation is to stick with task ID first, then if we can resolve it, we would put the task number in. Flip-flopping is discouraged in SE. We want to ensure that for every release, the user guide matches what the user can do. So, we should not be documenting features that are not in the release!

Comment (advaypal) [reply docs/UserGuide.md:65]: Hmm

**Pull request merged**